src/dataServices/dynamoDbUtil.ts

```diff
@@ -5,13 +5,22 @@
 
 import { clone, generateMeta } from 'fhir-works-on-aws-interface';
 import flatten from 'flat';
-import { SEPARATOR } from '../constants';
+import { DDB_HASH_KEY_SEPARATOR, SEPARATOR } from '../constants';
 import DOCUMENT_STATUS from './documentStatus';
 
 export const DOCUMENT_STATUS_FIELD = 'documentStatus';
 export const LOCK_END_TS_FIELD = 'lockEndTs';
 export const VID_FIELD = 'vid';
 export const REFERENCES_FIELD = '_references';
+export const TENANT_ID_FIELD = '_tenantId';
+export const INTERNAL_ID_FIELD = '_id';
+
+export const buildHashKey = (id: string, tenantId?: string): string => {
+    if (tenantId !== undefined) {

```

Comment:
should this just be truthy? is an empty string `""` a valid tenantID?
```suggestion
    if (tenantId) {

```

Reply:
updated.

truthy is fine here. empty string is not a valid tenantId. tenantId is validated on the router.